docs/developers-guide/debug.md

```diff
@@ -133,87 +132,131 @@ Example:
-        at /usr/local/go/src/runtime/asm_amd64.s:1333
+     0  0x00000000004ead9a in syscall.Syscall6
+        at /usr/local/go/src/syscall/asm_linux_amd64.s:53
+     1  0x0000000000524f55 in os.(*Process).blockUntilWaitable
+        at /usr/local/go/src/os/wait_waitid.go:31
+     2  0x00000000005194ae in os.(*Process).wait
+        at /usr/local/go/src/os/exec_unix.go:22
+     3  0x00000000005180a1 in os.(*Process).Wait
+        at /usr/local/go/src/os/exec.go:125
+     4  0x00000000007d870f in os/exec.(*Cmd).Wait
+        at /usr/local/go/src/os/exec/exec.go:501
+     5  0x0000000000d6c2fa in github.com/opencontainers/runc/libcontainer.  (*initProcess).wait
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/process_linux.go:655
+     6  0x0000000000d6c43f in github.com/opencontainers/runc/libcontainer.(*initProcess).terminate
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/process_linux.go:668
+     7  0x0000000000d89f35 in github.com/opencontainers/runc/libcontainer.(*initProcess).start.func1
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/process_linux.go:353
+     8  0x0000000000d6bace in github.com/opencontainers/runc/libcontainer.(*initProcess).start
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/process_linux.go:592
+     9  0x0000000000d3f3ae in github.com/opencontainers/runc/libcontainer.(*linuxContainer).start
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/container_linux.go:390
+    10  0x0000000000d3e426 in github.com/opencontainers/runc/libcontainer.(*linuxContainer).Start
+        at /root/nestybox/sysbox/sysbox-runc/libcontainer/container_linux.go:287
+    11  0x0000000000e1da2e in main.(*runner).run
+        at /root/nestybox/sysbox/sysbox-runc/utils_linux.go:383
+    12  0x0000000000e1f08f in main.startContainer
+        at /root/nestybox/sysbox/sysbox-runc/utils_linux.go:553
+    13  0x0000000000e1f78c in main.glob..func2
+        at /root/nestybox/sysbox/sysbox-runc/create.go:108
+    14  0x0000000000bac838 in github.com/urfave/cli.HandleAction
+        at /go/pkg/mod/github.com/urfave/cli@v1.22.1/app.go:523
+    15  0x0000000000bade00 in github.com/urfave/cli.Command.Run
+        at /go/pkg/mod/github.com/urfave/cli@v1.22.1/command.go:174
+    16  0x0000000000baa123 in github.com/urfave/cli.(*App).Run
+        at /go/pkg/mod/github.com/urfave/cli@v1.22.1/app.go:276
+    17  0x0000000000e11880 in main.main
+        at /root/nestybox/sysbox/sysbox-runc/main.go:145
+    18  0x000000000043ad24 in runtime.main
+        at /usr/local/go/src/runtime/proc.go:203
+    19  0x000000000046c0b1 in runtime.goexit
+        at /usr/local/go/src/runtime/asm_amd64.s:1357
+    (dlv)
+    ```
+
+-   Configure the source-code path:
+
+    If no code is displayed while iterating through the back-trace frames, then
+    ensure that Delve is properly configured with the path to your Sysbox
+    workspace.
+
+    By default, Sysbox expects its source-code to be at `/root/nestybox/sysbox`.
```

Comment:
This part I don't get; Sysbox does not expect this; it's only that when Sysbox is build inside the build container, it exists as this path.

Reply:
Yeah, didn't explain myself well there, will rewrite as below ...

```
Sysbox compilation process is carried out inside a docker container. In order to do this, we bind-mount the user's Sysbox workspace (i.e. "sysbox" folder) into this path within the container: `/root/nestybox/sysbox`. Golang compiler includes this path into the generated Sysbox binaries. Thereby, if you are debugging Sysbox daemon in your host system, unless your workspace path fully matches the one above (unlikely), Delve will not be able to display the Sysbox source-code.

The typical solution in these cases is to modify Delve's configuration to replace the containerized path with the one in your local environment.
```